db/fill-db.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+# Simple script to ensure db and table creation before filling up
```

Comment:
The idea of this script is, since official postgreSQL image entrypoint loads `.sh` and `.sql` files but with no particular order, ensure we setup db and tables before filling them up with mocked data or wathever we need